test/wt_hang_analyzer/wt_hang_analyzer.py

```diff
@@ -0,0 +1,616 @@
+#!/usr/bin/env python
+"""Hang Analyzer module.
+
+A prototype hang analyzer for Evergreen integration to help investigate test timeouts
+
+1. Script supports taking dumps, and/or dumping a summary of useful information about a process
+2. Script will iterate through a list of interesting processes,
+    and run the tools from step 1. The list of processes can be provided as an option.
+
+Currenty only supports Linux. There are two issues with the MacOS and windows implementation:
```

Comment:
Typo: `currenty => currently`

Reply:
'windows' should be Windows.
'implementation' should be plural.

Reply:
Fixed